server/libs/objectHelpers.js

```diff
@@ -0,0 +1,14 @@
+/**
```

Comment:
I don't like how this file is a duplicate from the `src` folder - we really need to look into a proper way of sharing files.

Reply:
Potentially could turn it into a Package, but for a very specific use case...